tests/formats/test_pkg.py

```diff
@@ -4,20 +4,15 @@
 from mercury_engine_data_structures.formats.pkg import Pkg
 from mercury_engine_data_structures.game_check import Game
 
-_EMPTY_DREAD_PKG = (b'|\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
-                    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
-                    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
-                    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
-                    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
-                    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
-                    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00'
-                    b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00')
+_EMPTY_DREAD_PKG = (b'\x08\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00')
 
 
 def test_compare_dread(dread_path):
-    parse_and_build_compare(
-        Pkg.construct_class(Game.DREAD), Game.DREAD, dread_path.joinpath("packs/system/system.pkg")
-    )
+    pkg_files = [f for f in dread_path.rglob("*.pkg")]
```

Comment:
Preferable for having the list of files in the code, and then parametrizing the test.

Reply:
Also add test for SR files.

Reply:
Dread has 47 files and SR has 368 files.
For SR it is quite easy: As we never figured out how to rebuilt the pkgs, every test will fail except the empty pkgs. I can create a PR for my funny code which can parse all files except 11 from SR if you like but it doesn't make sense in this PR.

Reply:
Dunno if you really want me to explicitly write down all these files but I did it now how it is done for most other tests with the following + parametrizing the test:
```python
all_dread_pkg = [name for name in dread_data.all_name_to_asset_id().keys()
                   if name.endswith(".pkg")]
```

Added a test for SR but skipped for now because the already mentioned reason.